stampit.js

```diff
@@ -8,17 +8,23 @@
  **/
 'use strict';
 var forEach = require('mout/array/forEach');
-var mixIn = require('mout/object/mixIn');
-var merge = require('mout/object/merge');
 var map = require('mout/array/map');
 var forOwn = require('mout/object/forOwn');
-var mixInChain = require('./mixinchain.js');
 var slice = [].slice;
 
+var mixer = require('./mixer');
```

Comment:
Why do I have to look at 3 different files to figure out the logic you're setting up in 3 lines of code? =)